packages/electrumAnalyzer/electrumAnalyzer.0.3.4/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "David Chemouil <david.chemouil+electrum@onera.fr>"
+authors: ["David Chemouil" "Julien Brunel" "Denis Kuperberg"]
+homepage: "https://forge.onera.fr/projects/electrum"
+bug-reports: "https://forge.onera.fr/projects/electrum"
+license: "GPL-3+"
+dev-repo: "https://git.onera.fr/electrum.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["rm" "-f" "%{bin}%/electrumAnalyzer"] {os = "darwin" | os = "linux"}
+]
+depends: [
+  "batteries" {build}
+  ("cmdliner" {build & = "0.9.7"} | "cmdliner" {build & = "0.9.6"} |
+   "cmdliner" {build & = "0.9.5"})
+  "menhir" {build & >= "20150720"}
+  "ocamlfind" {build & >= "1.5.5"}
+  "pprint" {build}
```

Comment:
On Mon, 2 Nov 2015 02:24:58 -0800, David Chemouil wrote:
> 
> In packages/electrumAnalyzer/electrumAnalyzer.0.3.4/opam:
> 
> > +dev-repo: "https://git.onera.fr/electrum.git";

I do not think it is great that this link requires authentication (it will only frustrate users who try to click on the link).  Maybe you want to use the Github mirror instead?